core/src/test/java/nl/nn/adapterframework/configuration/digester/ValidateAttributeRuleTest.java

```diff
@@ -310,4 +411,24 @@ public static class ConfigWarningTestClass implements INamedObject {
 	public static class DeprecatedTestClass {
 		private @Getter @Setter String name;
 	}
+
+	private static class SimpleTestObjectCreationFactory implements ObjectCreationFactory<Object> {
```

Comment:
Waar wordt deze methode gebruikt? Ik kan hem nergens terugvinden?

Reply:
> Waar wordt deze methode gebruikt? Ik kan hem nergens terugvinden?

Oh die kan inderdaad weer weg; die werd gebruikt in een oude versie van de test waarin ik de Digester helemaal zelf configureerde. Bij het opruimen daarvan over het hoofd gezien, bedankt!